### PR TITLE
[lemmas] Small cleanups

### PR DESCRIPTION
**Pull request by ejgallego**
These are some small cleanups that I'm pushing to reduce the delta of my internal trees, which are taking longer to sort out than what I thought.

### CODE REVIEW

File: vernac/lemmas.ml [hunk 1]
```diff
@@ -395,10 +370,38 @@ let save_lemma_admitted ~(lemma : t) : unit =
...
+     { const with proof_entry_body =
+        Future.chain const.proof_entry_body
+        (fun ((body, ctx), eff) ->
+          match Constr.kind body with
+          | Fix ((nv,0),(_,_,fixdefs as fixdecls)) ->
+(*      let possible_indexes =
+        List.map2 (fun i c -> match i with Some i -> i | None ->
+          List.interval 0 (List.length ((lam_assum c))))
+          lemma_guard (Array.to_list fixdefs) in
+*)
```
Comment (SkySkimmer) [inline vernac/lemmas.ml]: Seems like this was commented out in 93a5f1e03e29e375be69a2361ffd6323f5300f86, maybe it's time to remove the comment

**Review by SkySkimmer — APPROVED** [review]
Breaks equations

**Comment by ejgallego** [comment]
> Breaks equations

Yup, need to drop a commit actually, will clean the comment up too.

**Comment by SkySkimmer** [comment]
Is this ready?

**Comment by ejgallego** [comment]
> Is this ready?

I'd say so.

**Pull request merged**